### PR TITLE
Payload Service

### PR DESCRIPTION
**Pull request by JCrawsh**
Created Post payload endpoint and added Transaction types as an enum.

### CODE REVIEW

File: xumm/models/payload_request.go [hunk 1]
```diff
@@ -10,6 +10,10 @@ type XummPostPayload struct {
 	CustomMeta *XummCustomMeta         `json:"custom_meta,omitempty"`
 }
 
+type JsonTransaction struct {
```
Comment (bmurphy333) [inline xumm/models/payload_request.go:13]: Can remove this struct now

Comment (JCrawsh) [reply xumm/models/payload_request.go:13]: Yes good point!

File: xumm/payload/transaction_types.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+type TransactionTypeError struct {
+}
+
+func (e *TransactionTypeError) Error() string {
+	return "Invalid transaction type provided."
+}
+
+const (
+	// Invalid TransactionType = iota
```
Comment (bmurphy333) [inline xumm/payload/transaction_types.go:19]: remove this line

**Review by joshwalong — APPROVED** [review]
lgtm

**Review by bmurphy333 — APPROVED** [review]

**Pull request merged**